src/components/modal/confirm_modal.stories.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+
+import { moveStorybookControlsToCategory } from '../../../.storybook/utils';
+import { BUTTON_COLORS } from '../../themes/amsterdam/global_styling/mixins';
+import {
+  CANCEL_BUTTON,
+  CONFIRM_BUTTON,
+  EuiConfirmModal,
+  EuiConfirmModalProps,
+} from './confirm_modal';
+
+import { EuiButton } from '../button';
+
+const meta: Meta<EuiConfirmModalProps> = {
+  title: 'Layout/EuiConfirmModal',
+  component: EuiConfirmModal,
+  argTypes: {
+    // TODO: the `maxWidth` prop takes many different types (bool, string, number)
+    title: { control: { type: 'text' } },
+    confirmButtonText: { control: { type: 'text' } },
+    cancelButtonText: { control: { type: 'text' } },
+    defaultFocusedButton: {
+      control: { type: 'radio' },
+      options: [undefined, CONFIRM_BUTTON, CANCEL_BUTTON],
+    },
+    buttonColor: { control: { type: 'select' }, options: BUTTON_COLORS },
```

Comment:
I know this isn't the default, but can we also add `isLoading: false` here so that clicking to toggle the boolean only takes 1 click as opposed to 2?
```suggestion
    buttonColor: { control: { type: 'select' }, options: BUTTON_COLORS },
    isLoading: false,
```

Reply:
Do we maybe want to generally do this for boolean values in stories to setup the control properly? Considering that `undefined` results in falsy.

Reply:
Personally as the dev clicking these toggle buttons I think so! It'll save me _tens_ of clicks over the years 🤣

Reply:
🗒️ Maybe something to check as part of the whole "do we need a custom addon to handle more control types" topic.